code/__defines/_compile_options.dm

```diff
@@ -1,3 +1,9 @@
 // The default value for all uses of set background. Set background can cause gradual lag and is recommended you only turn this on if necessary.
 // 1 will enable set background. 0 will disable set background.
 #define BACKGROUND_ENABLED 0
+
+#define REQUIRED_DM_VERSION 513
+
+#if DM_VERSION < REQUIRED_DM_VERSION
+#warn Nebula is not tested on BYOND versions older than 513. The code may not compile, and you get to keep both halves if it breaks.
```

Comment:
Should really be 'and if it does compile it may have severe problems' at the end there, current phrasing is a bit confusing.